src/components/Navbar/Navbar.component.test.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+import { Navbar } from './Navbar.component'
+
+test('Properly render navbar', () => {
+    render(<Navbar />)
+
+    const searchBar = screen.queryByText(/Search/i)
+
+    expect(searchBar).toBeInTheDocument()
+})
```

Comment:
Good test 👌 

You also can test the existence of the menu element by testid (because is an icon)